pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -107,26 +108,33 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	err = r.Get(context.TODO(), request.NamespacedName, storage)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			log.V(2).Info("unable to get migstorage, not found", "migstorage", request.String(), "error", err.Error())
 			return reconcile.Result{}, nil
 		}
-		log.Trace(err)
+		log.Trace(fmt.Errorf("unable to get migstorage: %#v", err), "migstorage", request.String())
 		return reconcile.Result{Requeue: true}, nil
 	}
 
 	// Report reconcile error.
 	defer func() {
-		if err == nil || errors.IsConflict(err) {
+		switch {
+		case err == nil:
+			log.V(4).Info("migstorage reconcile successful", "migstorage", request.String())
+			return
+		case errors.IsConflict(err):
+			log.Error(fmt.Errorf("unable to get migstorage: %#v", err), "migstorage", request.String())
```

Comment:
Might be missing something, but this log message seems to mislead the user into thinking we couldn't `get` the migstorage, when in reality we hit a `ConflictError`.

Reply:
+1, I can reword the error message to reflect the update failure